.github/workflows/test_and_build.yml

```diff
@@ -101,7 +101,7 @@ jobs:
         with:
           targetPlatform: ${{ matrix.targetPlatform }}
           allowDirtyBuild: true
-          buildMethod: UnityBuildTools.UnityBuilderAction.Perform${{ matrix.targetDevice }}Build
+          buildMethod: GVR.Editor.UnityBuildTools.UnityBuilderAction.Perform${{ matrix.targetDevice }}Build
```

Comment:
Great. Thanks for the namespace. But you didn't add it to the actual file. This will not run afaik.